front/lib/triggers/temporal/webhook/activities.ts

```diff
@@ -168,6 +168,22 @@ export async function runTriggerWebhookActivity({
       );
       throw new TriggerNonRetryableError(errorMessage);
     }
+
+    const blacklist = WEBHOOK_PRESETS[webhookSource.provider].event_blacklist;
+    if (blacklist && blacklist.includes(receivedEventValue)) {
+      // Silently ignore blacklisted events
+      await webhookRequest.markAsProcessed();
+      logger.info(
+        {
+          workspaceId,
+          webhookRequestId,
+          provider: webhookSource.provider,
+          eventValue: receivedEventValue,
+        },
+        "Webhook event is blacklisted, ignoring."
+      );
+      return;
+    }
```

Comment:
I think it should be before the previous throw because we will always match this for ping events:

```
!webhookSource.subscribedEvents.includes(receivedEventValue)
```